src/components/Items/ProfileContact.js

```diff
@@ -1,10 +1,14 @@
 import * as React from 'react';
 import Mi from 'react-native-vector-icons/MaterialIcons';
 import {Image, Pressable, StyleSheet, Text, View} from 'react-native';
-
+import PropTypes from 'prop-types';
 import MemoIc_Checklist from '../../assets/icons/Ic_Checklist';
 import {COLORS} from '../../utils/theme';
-import {fonts, normalize, normalizeFontSize} from '../../utils/fonts';
+import {normalize, normalizeFontSize} from '../../utils/fonts';
+import {colors} from '../../utils/colors';
+import dimen from '../../utils/dimen';
+import {ANONYMOUS_USER} from '../../hooks/core/constant';
+import {isContainUrl} from '../../utils/Utils';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statements have been updated. The `fonts` import has been removed and replaced with `normalize, normalizeFontSize` from `../../utils/fonts`. Also, the `colors` import from `../../utils/colors` and `dimen` from `../../utils/dimen` have been added. These changes seem to be necessary for the new functionality and there doesn't appear to be any issues with them.

```diff
- import {fonts, normalize, normalizeFontSize} from '../../utils/fonts';
+ import {normalize, normalizeFontSize} from '../../utils/fonts';
+ import {colors} from '../../utils/colors';
+ import dimen from '../../utils/dimen';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->